src/pkg/docs/model/attributes.go

```diff
@@ -5,14 +5,18 @@ import "hms/gateway/pkg/indexer/ehrIndexer"
 type Attribute = uint8
 
 const (
-	AttributeID             Attribute = 1
-	AttributeIDEncr         Attribute = 2
-	AttributeKeyEncr        Attribute = 3
-	AttributeDocBaseUIDHash Attribute = 4
-	AttributeDocUIDEncr     Attribute = 5
-	AttributeDealCid        Attribute = 6
-	AttributeMinerAddress   Attribute = 7
-	AttributeContentEncr    Attribute = 8
+	AttributeID              Attribute = 1
+	AttributeIDEncr          Attribute = 2
+	AttributeKeyEncr         Attribute = 3
+	AttributeDocBaseUIDHash  Attribute = 4
+	AttributeDocUIDEncr      Attribute = 5
+	AttributeDealCid         Attribute = 6
+	AttributeMinerAddress    Attribute = 7
+	AttributeContent         Attribute = 8
+	AttributeContentEncr     Attribute = 9
+	AttributeDescriptionEncr Attribute = 10
+	AttributePasswordHash    Attribute = 11
+	AttributeTimestamp       Attribute = 12
```

Comment:
you can use `iota` to enumerate `enum` values
```go
type Attribute = uint8

const (
	AttributeID Attribute = iota + 1
	AttributeIDEncr
	AttributeKeyEncr
	AttributeDocBaseUIDHash
	AttributeDocUIDEncr
	AttributeDealCid
	AttributeMinerAddress
	AttributeContent
	AttributeContentEncr
	AttributeDescriptionEncr
	AttributePasswordHash
	AttributeTimestamp
)
```

Reply:
I know, but we have same enum in contact 
https://github.com/bsn-si/IPEHR-blockchain-indexes/blob/develop/contracts/libraries/Attributes.sol
and we have to keep it in same order. 
I did it this way on purpose so that the numbering would be visible